### PR TITLE
Fixed mantissa threshold for hf8 overflow convert

### PR DESCRIPTION
**Pull request by egeor**

### CODE REVIEW

File: src/libxsmm_lpflt_quant.c [hunk 1]
```diff
@@ -576,7 +576,7 @@ LIBXSMM_API void libxsmm_rne_convert_fp32_hf8(const float* in, libxsmm_hfloat8*
       m = 0x7;
     /* overflow --> make it NaN */
     } else if ( (e_f32 > (f32_bias - f8_bias + 15)) ||
-                ((e_f32 == (f32_bias - f8_bias + 15)) && ( m_f32 > 0x600000))) {
+                ((e_f32 == (f32_bias - f8_bias + 15)) && ( m_f32 > 0x680000))) {
```
Comment (alheinecke) [inline src/libxsmm_lpflt_quant.c:579]: Why is a similar fix not needed for code gen?

**Review by alheinecke — COMMENTED** [review]
small question.

File: src/libxsmm_lpflt_quant.c (hunk 1 above)
Comment (egeor) [reply src/libxsmm_lpflt_quant.c:579]: We had decided to use only the routine with f32->f16 and f16->hf8 path for the codegen (this is is just there for future reference)

Comment (alheinecke) [reply src/libxsmm_lpflt_quant.c:579]: ACK.

**Review by alheinecke — APPROVED** [review]

**Comment by alheinecke** [comment]
btw, do we also need changes to our bias conversion routines?

**Comment by egeor** [comment]
> btw, do we also need changes to our bias conversion routines?

we do not have hf8 cvt with bias in libxsmm...

**Pull request merged**